src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import planet from '../assets/planet.png';
+
+const Header = () => (
+  <div className="panel-bg container">
+    <nav className="nav row">
+      <div className="col-8 row">
+        <img src={planet} alt="planet-logo" className="planet-logo col-4" />
+        <h1 className="title  col-8">Space Travelers Hub</h1>
+      </div>
+      <div className="col-4 pt-3">
+        <NavLink to="/" activeClassName="active-link ">
+          <span className="rockets-nav px-3 mx-3">Rockets</span>
+        </NavLink>
+        <NavLink to="/missions" activeClassName="active-link">
+          <span className="missions-nav px-3 mx-3">Missions</span>
+        </NavLink>
+        <NavLink to="/Profile" activeClassName="active-link">
+          <span className="profile-nav px-5 ">Profile</span>
+        </NavLink>
```

Comment:
- [ ] This requirement is half-implemented. We are supposed to show which page the user is currently on by underlining the active link. Please take another look at that specific requirement and make the needed adjustment 👇🏼:
---
> Create routes and view components (Rockets, Missions, My Profile, Dragons [only if your team has 3 members]). Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).